### DISCUSSION TITLE
Is there a doc that lists down all the breaking changes between the two versions?

### DISCUSSION THREAD

**Discussion by bashonregardless** [testing-library/user-event#965] [General]
I want to upgrade from v13.5.0 to v14.x.x.
Is there a doc that lists down all the breaking changes between the two versions?

**Comment by ph-fritsche** [comment]
You can find all breaking changes in the release notes: https://github.com/testing-library/user-event/releases/tag/v14.0.0

**Reply by bashonregardless** [reply]
Thank you @ph-fritsche
